shenyu-plugin/shenyu-plugin-hystrix/src/main/java/org/apache/shenyu/plugin/hystrix/HystrixPlugin.java

```diff
@@ -53,7 +54,9 @@ public class HystrixPlugin extends AbstractShenyuPlugin {
     protected Mono<Void> doExecute(final ServerWebExchange exchange, final ShenyuPluginChain chain, final SelectorData selector, final RuleData rule) {
         final ShenyuContext shenyuContext = exchange.getAttribute(Constants.CONTEXT);
         assert shenyuContext != null;
-        final HystrixHandle hystrixHandle = HystrixPluginDataHandler.CACHED_HANDLE.get().obtainHandle(CacheKeyUtils.INST.getKey(rule));
+        final HystrixHandle hystrixHandle = new HystrixHandle();
+        // Avoid modifying objects in the cache
+        BeanUtils.copyProperties(HystrixPluginDataHandler.CACHED_HANDLE.get().obtainHandle(CacheKeyUtils.INST.getKey(rule)), hystrixHandle);
```

Comment:
why need used BeanUtils.copyPropertie？

Reply:
because the following `hystrixHandle.setGroupKey` will modify the data in the cache, causing other errors.

Reply:
maybe my current commit can solve the problem another way.

Reply:
Hi， in  shenyu-admin， Is the groupKey configured?

Reply:
yes 
![image](https://user-images.githubusercontent.com/38271111/183831468-f6985d7d-98aa-448a-b5d5-da929c48ff8b.png)